src/db/index.js

```diff
@@ -0,0 +1,61 @@
+const Sequelize = require('sequelize');
+const elasticsearch = require('elasticsearch');
+const { orderSchema, pairSchema } = require('./schemas');
+const { POSTGRES: { USER, PASSWORD, HOST }} = require('../../config');
+
+
+const sequelize = new Sequelize('orderBook', USER, PASSWORD, {
+  host: HOST,
+  dialect: 'postgres',
+  sync: { force: true },
+  syncOnAssociation: true,
+  pool: { maxConnections: 5, maxIdleTime: 30},
+});
+
+sequelize
+  .authenticate()
+  .then(() => {
+    console.log('Connection has been established successfully.');
+  })
+  .catch(err => {
+    console.error('Unable to connect to the database:', err);
+  });
+
+
+const Buy = sequelize.define('buy', orderSchema);
+const Sell = sequelize.define('sell', orderSchema);
+const Pair = sequelize.define('pair', pairSchema);
+
+Pair.sync().then(result => console.log(result));
+
+Buy.belongsTo(Pair, {as: 'pair'});
```

Comment:
Define two way relationship is probably better.

Reply:
Made requested changes